browse/services/dissemination/article_store.py

```diff
@@ -403,3 +408,24 @@ def _e_print(self,
         Returns `FileObj` if found, `None` if not."""
         src = self.sourcestore.get_src(arxiv_id, docmeta)
         return src if src is not None else "NO_SOURCE"
+
+
+    def _html(self, arxiv_id: Identifier, docmeta: DocMetadata, version: VersionEntry) -> FormatHandlerReturn:
+        """Gets the html src as submitted for the arxiv_id. Returns `FileObj` if found, `None` if not."""
+
+        if docmeta.source_format == 'html':#native html
+            path=ps_cache_html_path(arxiv_id, version.version)
+            files=self.objstore.list(path) 
+            file_list=list(files)
+            if len(file_list) >0:
+                return file_list
+            else:
+                return "NO_SOURCE"
```

Comment:
I don't like this use of sentinel strings.  Just return the file list, which will test as "False-y" if there are no files.

Same comment below (NO_HTML).